library/string.lisp

```diff
@@ -195,6 +195,16 @@ does not have that suffix."
       (lisp String (lst)
         (cl:coerce lst 'cl:string))))
 
+  (define-instance (Into (List String) String)
```

Comment:
is this the right canonical implementation?

Reply:
changed to `mconcat` and moved it fully into `Coalton` land

Reply:
Yes but why should the `String` of a list of strings be their concatenation? It sounds like a loose interpretation of what Into should mean.

Reply:
I get what you mean now, I may leave the cell definition then and scrap these two